README.md

```diff
@@ -0,0 +1,125 @@
+# Understanding a Temporal workflow as an implementation of the Actor Model
```

Comment:
Please capitalize Workflows when referring to a Workflow in the Temporal context

Reply:
As well as capitalizing Signal, Child/Parent Workflows